packages/cryptoplease_link/lib/src/handlers/moonpay_handler.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:convert';
+import 'dart:core';
+
+import 'package:crypto/crypto.dart';
+import 'package:cryptoplease_link/src/constants.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf_router/shelf_router.dart' as shelf_router;
+
+Future<Handler> moonpayHandler() async {
```

Comment:
Let's rather do it in a feature-first way, in the same way how it's done with `payments` feature (yeah, I know that it's a little bit inconsistent now, we should fix it later).